great_expectations/data_context/store/expectations_store.py

```diff
@@ -103,14 +128,14 @@ def gx_cloud_response_json_to_object_dict(response_json: Dict) -> Dict:
                 )
         else:
             suite_data = response_json["data"]
-        ge_cloud_suite_id: str = suite_data["id"]
-        suite_dict: Dict = suite_data["attributes"]["suite"]
-        suite_dict["id"] = ge_cloud_suite_id
 
-        # Temporary fork to account for pre-V1 configs
-        suite_dict.pop("ge_cloud_id", None)
+        # Cloud backend adds a default result format type of None, so ensure we remove it:
+        for expectation in suite_data.get("expectations", []):
+            if expectation.get("kwargs", {}).get("result_format", False) is None:
+                expectation["kwargs"].pop("result_format")
```

Comment:
this coverage check is incorrect; see `test_expectation_store.test_gx_cloud_response_json_to_object_dict['null_result_format']`

Reply:
@joshua-stauffer, it's possible I missed adding a coverage report upload for this test (or it isn't actually being run).

What test step is this running on? `could-tests` I would assume.